src/Symfony/Installer/NewCommand.php

```diff
@@ -356,6 +358,22 @@ private function cleanUp()
     {
         $this->fs->remove(dirname($this->compressedFilePath));
 
+        try {
+            $licenseFile = array($this->projectDir.'/LICENSE');
+            $upgradeFiles = glob($this->projectDir.'/UPGRADE*.md');
+            $changelogFiles = glob($this->projectDir.'/CHANGELOG*.md');
```

Comment:
Honestly, I'm not comfortable removing any `*.md` file as suggested. I prefer to be careful and remove the files we know for certain that should be removed (CHANGELOG, UPGRADE, LICENSE).